src/styles/blocks/button.scss

```diff
@@ -0,0 +1,22 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
```

Comment:
```suggestion
```
It is better to apply `box-sizing: border-box;` for all elements via `*` selector